common/lib/user_input.py

```diff
@@ -33,6 +33,7 @@ class UserInput:
     OPTION_DATERANGE = "daterange"  # a beginning and end date
     OPTION_DIVIDER = "divider"  # meta-option, divides related sets of options
     OPTION_FILE = "file"  # file upload
+    OPTION_FILES = "files"  # multiple files upload
```

Comment:
The only difference between "files" and "file" seems the `multiple` attribute in the HTML - could it be an option (like `requires`, `tooltip`, etc) instead?